Packages/cdms2/Lib/dataset.py

```diff
@@ -283,9 +283,10 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
     uri = string.strip(uri)
     (scheme,netloc,path,parameters,query,fragment)=urlparse.urlparse(uri)
     if scheme in ('','file'):
+        if uri[:7].lower() == "file://":
```

Comment:
@chaosphere2112 I thouught that urllib would have something .. but this code is fine.